homeassistant/components/generic/config_flow.py

```diff
@@ -66,8 +66,15 @@
     CONF_VERIFY_SSL: True,
 }
 
-SUPPORTED_IMAGE_TYPES = {"png", "jpeg", "gif", "svg+xml", "webp"}
+SUPPORTED_IMAGE_TYPES = {
+    "image/png",
+    "image/jpeg",
+    "image/gif",
+    "image/svg+xml",
+    "image/webp",
+}
 IMAGE_PREVIEWS_ACTIVE = "previews"
+SVG_REGEX = r"\s*(?:<\?xml\b[^>]*>[^<]*)?(?:<!--.*?-->[^<]*)*(?:<svg|<!DOCTYPE svg)\b"
```

Comment:
Not used anymore?

Reply:
Yep. I'm also reminded of https://stackoverflow.com/a/1732454/51685 ... 😁

Reply:
Agreed, done. afae319